v3/plugins/cdr/code-server/next/meta.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+publisher: cdr
+name: code-server
+version: next
+type: Che Editor
+displayName: code-server
+title: Visual Studio Code - Web (code-server)
+description: An open source distribution of Visual Studio Code as a cloud IDE
+icon: https://raw.githubusercontent.com/sr229/discord-open-source/master/logos/code-server.svg 
+category: Editor
+repository: https://github.com/cdr/code-server
+firstPublicationDate: "2019-09-08"
+spec:
+  endpoints:
+   -  name: "code-server"
+      public: true
+      targetPort: 9000
+      attributes:
+        protocol: http
+        type: ide
+  containers:
+   - name: code-server
+     image: "docker.io/chinodesuuu/coder:next"
```

Comment:
latest images are now on docker.io/codercom/code-server, which is why @ericwill suggested we could just remove this entirely and keep a single versioned version (3.6.2) until we move to the latest 3.7 one.

Reply:
he never suggested that in the PR, the PR doesn't mention that as well
Not written : never happened !